.github/workflows/automerge-orphans.yml

```diff
@@ -20,13 +20,14 @@ jobs:
           script: |
             const query = `query($owner:String!, $name:String!) {
               repository(owner:$owner, name:$name){
-                pullRequests(first: 100, states: OPEN){
+                pullRequests(first: 100){
```

Comment:
this file must entirely reflect https://github.com/asyncapi/.github/blob/master/.github/workflows/automerge-orphans.yml

Reply:
thanks will do